calc/tests/test_calc.py

```diff
@@ -1,5 +1,7 @@
-from calc import add
+from calc import add, subtract
 
 def test_add():
     assert add(1, 2) == 3
 
+def test_sub():
+    assert subtract(2, 1) == 1
```

Comment:
Maybe add a few more tests to check functionality especially with negative numbers.